odoo/addons/test_orm/tests/test_schema.py

```diff
@@ -226,519 +88,443 @@ def test_models_fields(self):
                             self.assertEqual(field_description['sortable'], field.type != 'binary')
 
 
-@tagged('at_install', '-post_install')  # LEGACY at_install
+@tagged('at_install', '-post_install')
 class TestSchema(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.model = cls.env['test_schema']
+        cls.table_name = cls.model._table
```

Comment:
This one is not very useful IMHO.  I would use `self.model._table` instead of `self.table_name`, so that it is very clear which model we consider.